src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,7 @@
+package org.example;
+
+public class Main {
```

Comment:
Why do you need this class?

Reply:
I am not sure where it came from to be fair 
Though it was included in the source directory